### PR TITLE
Start Port to 1.18.1

### PR DESCRIPTION
**Pull request by rocketstrong600**
just done some gradle stuff and started on a few things

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -158,15 +158,14 @@ dependencies {
...
         attributes([
-                "Specification-Title"     : "examplemod",
-                "Specification-Vendor"    : "examplemodsareus",
+                "Specification-Title"     : "FPVMod",
+                "Specification-Vendor"    : "gluecode",
                 "Specification-Version"   : "1", // We are version 1 of ourselves
                 "Implementation-Title"    : project.name,
                 "Implementation-Version"  : project.jar.archiveVersion,
-                "Implementation-Vendor"   : "examplemodsareus",
+                "Implementation-Vendor"   : "gluecode",
```
Comment (taylorgoolsby) [inline build.gradle:168]: thank you lol

**Comment by Amzd** [comment]
Hey could you leave some notes on what still needs to be done. Would make it easier to pick up for someone else in case you dont have time.

**Comment by rocketstrong600** [comment]
> ds to be done. Would make it easier to pi

well i am just making errors go away first then making it work. so error messages are still being done

**Comment by taylorgoolsby** [comment]
@rocketstrong600 Can you give me access to push to this branch?
Does this work: https://stackoverflow.com/a/49854013

**Comment by rocketstrong600** [comment]
> @rocketstrong600 Can you give me access to push to this branch? Does this work: https://stackoverflow.com/a/49854013

it is enabled already.

**Pull request merged**